arviz/plots/ecdfplot.py

```diff
@@ -46,26 +52,41 @@ def plot_ecdf(
     values : array-like
         Values to plot from an unknown continuous or discrete distribution.
     values2 : array-like, optional
-        Values to compare to the original sample.
+        deprecated: values to compare to the original sample. Instead use
+        `cdf=scipy.stats.ecdf(values2).cdf.evaluate`.
```

Comment:
I hope in a future PR to add an ECDF-(difference)plot option to `plot_ranks` and then recommend that here.